pyiron/atomistics/structure/atoms.py

```diff
@@ -1234,8 +1233,8 @@ def get_neighbors_by_distance(
             volume_per_atom = self.get_volume(per_atom=True)
             if id_list is not None:
                 volume_per_atom = self.get_volume() / len(id_list)
-            num_neighbors = int((1 + num_neighbors_estimate_buffer) *
-                                4. / 3. * np.pi * cutoff_radius ** 3 / volume_per_atom)
+            num_neighbors = max(1, int((1 + num_neighbors_estimate_buffer) *
+                                       4. / 3. * np.pi * cutoff_radius ** 3 / volume_per_atom))
```

Comment:
We could also just do `int(...)+1`

Reply:
Yes but find this a bit more intuitive and saves (even though only a little) computation time

Reply:
Is 1 a really solid estimation? I'm asking this because while for a large radius the estimation for the number of atoms from the density is a robust algorithm, there might be strong fluctuations for a very small radius. In other words, we know from a usual radial distribution function that the value becomes stable at high radii, the behaviour for small radii might become quite unstable. If taking a few atoms is still acceptable from the computational point of view, I might put 3 or 4.

Reply:
Good point!